packages/ts/react-crud/test/autoform.spec.tsx

```diff
@@ -39,7 +40,7 @@ describe('@hilla/react-crud', () => {
     function getExpectedValues(person: Person) {
       return (Object.entries(person) as ReadonlyArray<[keyof Person, Person[keyof Person]]>)
         .filter(([key]) => KEYS.includes(key))
-        .map(([, value]) => value);
+        .map(([, value]) => value.toString());
```

Comment:
This should return something that can be compared to field values, which will always be strings. So the numbers in the expected person object need to be converted into strings as well. I think this was working previously because binder was setting the value of `vaadin-text-field` to a `0` (the number, not the string), and the component doesn't have logic to sanitize that into a string. `vaadin-integer-field` and `vaadin-number-field` OTOH do have that sanitization.